NSS-Spontaneous-Spoonbills/Sprint2
# Cr employee details

## Link to Ticket

#3 

Closes: No ticket to close

## Description of Proposed Changes

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout crEmployeeDetails
```
1. pyman makemigrations WorkforceManagement
1. pyman migrate
1. pyman runserver

*DB Steps*

```
Must make at least:
1 employee
1 computer
1 department
1 training program
and training emp join table (connect employee to training program)
```

Once complete write database changes

1. Go to localhost:8000/WorkforceManagement/employees (Via the navbar button if you wish)
1. Click on employee name (It should now be a clickable link that redirects you)
1. It should display details for that specific employee

*DB Steps 2*

```
Must make at least:
1 more training program
and in training emp join table (connect employee to BOTH training programs)
```

Once complete write database changes

1. Go to localhost:8000/WorkforceManagement/employees 
1. Click on employee name
1. It should display details for that specific employee showing BOTH training programs under their name

*DB Steps 3*

```
Must make at least:
1 more employee
1 more training program
and in training emp join table (connect employee to a training program (or 2 or not))
```

Once complete write database changes

1. Go to localhost:8000/WorkforceManagement/employees 
1. Click on employee name
1. It should display details for that specific employee (Make sure the programs are still person specific )

## Impacted Areas in Application

List general components of the application that this PR will affect:

* Employee_View.py
* Employee_Detail.html
* Employee_List.html
* WorkforceManagement/urls.py

## Mentions @username

@MrErin @JacobAndrewSmith92 @binauralbeat @obscure-shadow 

## Testing


## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
